tasks/ast_codegen/src/generators/ast_builder.rs

```diff
@@ -309,6 +347,139 @@ impl ToTokens for Param {
     }
 }
 
+#[derive(Debug)]
+struct DocComment<'p> {
```

Comment:
It'll be nice if we move this type to somewhere else so we can use it across the board. I haven't spent time documenting the generated code but this is a nice way to do it.

Reply:
Where would you like it moved? And could this be done in a following PR?